packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -308,7 +336,9 @@ export function createTestEsCluster<
      * in this list.
      */
     getHostUrls(): string[] {
-      return this.ports.map((p) => format({ ...esTestConfig.getUrlParts(), port: p }));
+      return this.ports.map((p) =>
+        format({ ...esTestConfig.getUrlParts(ssl), port: p, protocol: ssl ? 'https' : 'http' })
```

Comment:
@elastic/kibana-operations I have to override protocol here because TEST_ES_URL doesn't have the correct protocol. So although this works it feels a bit hacky.